### DISCUSSION TITLE
License issue?

### DISCUSSION THREAD

**Discussion by yanghanlin** [adirh3/Fluent-Search#649] [Q&A] [answered]
The installer of Fluent Search (Exe distribution, version 0.9.92.28, nightly channel) seems to indicate that Fluent Search is licensed under [GPL v3](https://www.gnu.org/licenses/gpl-3.0.html) (see screenshot below), which requires the source code to be publicly available.

![Installer screenshot](https://user-images.githubusercontent.com/8692892/183591294-aa9b22c7-7024-4a5b-92c1-15d87228b7b7.png)

However, as far as I know, Fluent Search is not open source software, so it might not be suitable to be licensed under GPL without source code available. I think the author can either publicly release the source code or, if Fluent Search does not plan to be open sourced, replace the license with an EULA.

Anyway, I would like to express my gratitude for the author's work. Thanks for creating this excellent launcher!

**Comment by adirh3** [comment, marked as answer]
Thank you @YangHanlin! will change it.
